app/models.py

```diff
@@ -31,19 +31,44 @@ class User(Base):
     __tablename__ = "users"
 
 
+class Location(Base):
+    id = Column(Integer, primary_key=True, index=True)
+
+    osm_id = Column(BigInteger)
+    osm_type = Column(ChoiceType(PriceLocationOSMType))
```

Comment:
Not directly related to this PR, but we could rename PriceLocationOSMType into LocationOSMType (it doesn't add anything to add "Price")

Reply:
done here https://github.com/openfoodfacts/open-prices/pull/35/commits/2637cf778fed400dcf390ac4330b782deefdd7bb